.ruby-version

```diff
@@ -1 +1,3 @@
-2.2.3
+2.1.7
+# DO NOT UPGRADE past 2.1.X until we resolve the following in Govspeak:
```

Comment:
Turns out that the comment is triggering a false positive on the Icinga check for version mismatches. I've muted them for now as we should be able to upgrade Whitehall to 2.2.x now that govspeak 3.5.1 is available.